nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/cache/CacheService.java

```diff
@@ -42,6 +42,17 @@
      * @param size the maximum number of elements
      * @param timeout the entry timeout (in minutes)
      * @since 8.2
+     * @deprecated since 9.3, seems unused, use {@link #registerCache(String)} instead.
      */
+    @Deprecated
     public void registerCache(String name, int size, int timeout);
```

Comment:
To discuss if we deprecate this method, as we are not using it anymore.

Reply:
Yes it's ok, it's only used internally. Third-party code should use extension points instead.